tests/native/para-native.json

```diff
@@ -22,15 +22,6 @@
   },
   "parachains": [
     {
-      "genesis": {
-        "runtime": {
-          "runtime_genesis_config": {
-            "balances": {
-              "balances": {% include "../funded-accounts.json" %}
-            }
-          }
-        }
-      },
```

Comment:
where will the pre-funded accounts come from?

Reply:
oh I guess Versi does that in a different way? I had forgotten about that for a second

anyways, how do we make sure `sender` is using the right accounts?

Reply:
Versi does it differently indeed - the chain spec is basically already populated with the pre funded accounts. 

Regarding the zombienet side, let's add the configuration necessary once it's supported in zombienet. The previous configuration kept throwing "Bad configuration", so I removed it.